packages/forklift-console-plugin/src/modules/Providers/views/create/components/EsxiProviderCreateForm.tsx

```diff
@@ -44,14 +44,8 @@ export const EsxiProviderCreateForm: React.FC<EsxiProviderCreateFormProps> = ({
 
   const initialState = {
     validation: {
-      url: {
```

Comment:
Nice fix :+1:, we should replace all the places we use static constants for initial state with values we get dynamicly from the validators.